### DISCUSSION TITLE
Pairdrop GitHub problems

### DISCUSSION THREAD

**Discussion by marcusdoc** [tteck/Proxmox#2516] [Q&A]
I'm not sure if this impacts the proxmox script but the there is a problem with the GitHub repository with Pairdrop.

https://www.reddit.com/r/selfhosted/comments/1au269o/what_happened_to_the_pairdrop_github/

**Comment by tteck** [comment]
Removed from the website, thanks!

**Reply by flashbackpa** [reply]
Hi, thanks for your work, I guess there's a new repo for Pairdrop:
https://github.com/Ecalose/PairDrop

**Reply by tteck** [reply]
That's a fork. I don't _think_ it's the official repository.

**Comment by tteck** [comment]
> I changed my password and activated 2FA since.

He didn't use 2FA, wow.

**Comment by marcusdoc** [comment]
That is a bit odd that he didn't use 2AF

... thanks again for the scripts! your scripts make an integral part of my homelab
________________________________

**Comment by oliv99** [comment]
may be a good alternative
https://github.com/localsend/localsend

**Reply by marcusdoc** [reply]
"LocalSend" is app based; communicating from one app to another app.
I tried it on Windows 10 and found that it was slow to identify the other apps (might be a firewall issue).

This is where Pairdrop shined; no apps; no bloat; not nonsense ... just one webpage communicating with another webpage through a server.

**Comment by manuelkue** [comment]
The Pairdrop github repo is available again:
https://github.com/schlagmichdoch/PairDrop

See https://github.com/fm-sys/snapdrop-android/issues/356#issuecomment-2009167668
